website/docs/r/dns_linked_zone.html.markdown

```diff
@@ -0,0 +1,62 @@
+---
+subcategory: "DNS Services"
+layout: "ibm"
+page_title: "IBM : dns_linked_zone"
+description: |-
+  Manages IBM Private DNS linked zone.
+---
+
+# ibm_dns_linked_zone
+
+The DNS linked zone resource allows users to request and manage linked zones. 
+
+
+## Example usage
+
+```
+resource "ibm_dns_linked_zone" "test" {
+  name          = "test_dns_linked_zone"
+  instance_id   = "****************************"
+  description   = "seczone terraform plugin test"
+  owner_instance_id = "**************************"
+  owner_zone_id = "************************"
+  label         = "test"
+}
+
+data "ibm_dns_linked_zone" "test-lz" {
+  depends_on  = [ibm_dns_linked_zone.test]
+  instance_id   = "***********************"
+}
+
+output "ibm_dns_linked_zone_id" {
+  value = data.ibm_dns_linked_zone.test-lz.instance_id
+}
+
+```
+
+## Argument reference
+Review the argument reference that you can specify for your resource. 
+
+- `instance_id` - (Required, String) The unique identifier of a service instance.
```

Comment:
The arguments listed here is mismatch with what we defined in schema (description, owner_instance_id,...so on ar e missing)